lib/psi.rb

```diff
@@ -1213,14 +1222,34 @@ def processTBD(os_model, io_f, schema_f, psi_set)
   end
 
   # Assign thermal bridging heat loss [in W/K] to each deratable surface.
-  surfaces.values.each do |surface|
+  surfaces.each do |id, surface|
     next unless surface.has_key?(:edges)
     surface[:heatloss] = 0
     surface[:edges].values.each do |edge|
       surface[:heatloss] += edge[:psi] * edge[:length]
     end
   end
```

Comment:
Oops. Forgot to switch back to values; id is only used here sometimes for testing.